grpc.go

```diff
@@ -56,7 +56,7 @@ var (
 	sysfsCPUOnlinePath     = "/sys/devices/system/cpu"
 	sysfsMemOnlinePath     = "/sys/devices/system/memory"
 	sysfsConnectedCPUsPath = filepath.Join(sysfsCPUOnlinePath, "online")
-	sysfsCpusetPath        = "/sys/fs/cgroup/cpuset"
+	sysfsCpusetPath        = cgroupPath + "/cpuset"
```

Comment:
Does this work ? I mean in case of the agent is run as init, do you actually get `sysfdCpusetPath` containing the value `/proc/cgroup/cpuset` ?
I am asking because I thought the process would load every global variable in memory, meaning that at this time, it would have `cgroupPath = "/sys/fs/cgroup"` and therefore `sysfsCpusetPath = "/sys/fs/cgroup/cpuset"`.
My understanding is that `initAgentAsInit()` will get called later and the modification of the variable will only affect `cgroupPath`, but `sysfsCpusetPath` will remain unchanged.

Reply:
yep it works, maybe because `initAgentAsInit` is called before using any function in `grpc.go`
I need to change this since this is not a good way to initialize a variable
good catch! thanks